### PR TITLE
LGAT module pulled from most recent OpenPBTA repo

### PR DESCRIPTION
**Pull request by runjin326**
#### What scientific question is your analysis addressing?
Molecular subtyping module LGAT were modified in OpenPBTA. 
This PR pulled the most up-to-date OpenPBTA LGAT subtyping module.

#### What was your approach?
1. `molecular-subtyping-LGAT` from OpenPBTA repo were copied to OpenPedCan repo.
2. The following changes were made:
- `pbta-histologies-base.tsv` was changed to `histologies.tsv`
- Instead of taking `pbta-snv-consensus-mutation.maf.tsv.gz` and `pbta-snv-scavenged-hotspots.maf.tsv.gz`, an already combined file `snv-consensus-plus-hotspots.maf.tsv.gz` was used
- `pbta-sv-manta.tsv.gz` to `sv-manta.tsv.gz`
- `pbta-cnv-consensus.seg.gz` to `cnv-consensus.seg.gz`
- `consensus_seg_annotated_cn_autosomes.tsv.gz` to `consensus_wgs_plus_cnvkit_wxs.tsv.gz` (the combined file was used since that is the file in our current release - there is a step calling only WGS samples so using the combined file would not affect results)
- `00-v17-LGAT-select-pathology-dx.Rmd` were renamed to `00-v9-LGAT-select-pathology-dx.Rmd` to match OpenPedCan release number. 

#### What GitHub issue does your pull request address?
https://github.com/PediatricOpenTargets/ticket-tracker/issues/98

#### Which areas should receive a particularly close look?
1. Please check to see whether all the updates in OpenPBTA were captured. 
2. We currently don't have OpenPedCan `histologies-base.tsv` so I just used `histologies.tsv` - is this ok? 
3. I added `00-v17-LGAT-select-pathology-dx.Rmd` to the run script which is actually not necessary - should we keep that?

#### Is there anything that you want to discuss further?
No.

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes. 

#### What types of results are included (e.g., table, figure)?
Files in the modules are changed as discussed above.
The module was re-ran without issues. 

#### What is your summary of the results?
The additional path free text and path diagnosis shown to be different in this PR captured what is expected. 

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

**Comment by runjin326** [comment]
@jharenza, names changed and re-ran now :)

File: analyses/molecular-subtyping-LGAT/01-subset-files-for-LGAT.Rmd [hunk 1]
```diff
@@ -69,13 +69,33 @@ path_dx_list <- jsonlite::fromJSON(
 # clinical file
 clinical <- read_tsv(file.path(root_dir, 
                                "data",
-                               "pbta-histologies-base.tsv"), 
+                               "histologies.tsv"), 
                      guess_max = 10000)
```
Comment (jharenza) [inline analyses/molecular-subtyping-LGAT/01-subset-files-for-LGAT.Rmd]:
```suggestion
                     guess_max = 100000)
```

File: analyses/molecular-subtyping-LGAT/04-LGAT-compile-subtypes.Rmd [hunk 2]
```diff
@@ -44,40 +44,53 @@ lgat_metadata_file <- file.path("lgat-subset", "lgat_metadata.tsv")
...
+            "lgat_subtyping_path_dx_strings.json")
+)
+
 # clinical file
 lgat_manifest <- read_tsv(file.path(root_dir, 
                                     "data",
-                                    "pbta-histologies-base.tsv"), 
+                                    "histologies.tsv"), 
                           guess_max = 10000) %>%
   filter(Kids_First_Biospecimen_ID %in% bsid_to_include) %>%
```
Comment (jharenza) [inline analyses/molecular-subtyping-LGAT/04-LGAT-compile-subtypes.Rmd]:
```suggestion
                                    "histologies.tsv"), 
                          guess_max = 100000) %>%
  filter(Kids_First_Biospecimen_ID %in% bsid_to_include) %>%
```

File: analyses/molecular-subtyping-LGAT/README.md [hunk 3]
```diff
@@ -82,7 +83,8 @@ The files in the `lgat-subset` were generated via `01-subset-files-for-LGAT.R` u
 ### Inputs from data download
 
 * `pbta-histologies.tsv`: is used to subset samples according to [the criteria above](#inclusion-exclusion-criteria)
```
Comment (jharenza) [inline analyses/molecular-subtyping-LGAT/README.md]:
```suggestion
* `histologies.tsv`: is used to subset samples according to [the criteria above](#inclusion-exclusion-criteria)
```

File: analyses/molecular-subtyping-LGAT/README.md [hunk 4]
```diff
@@ -82,7 +83,8 @@ The files in the `lgat-subset` were generated via `01-subset-files-for-LGAT.R` u
...
-* `pbta-snv-consensus-mutation.maf.tsv.gz`: 
+* `pbta-snv-consensus-mutation.maf.tsv.gz`: from `snv-callers` module which gathers calls that are present in all 3 callers (strelka2,mutect2 and lancet) 
+* `pbta-snv-scavenged-hotspots.maf.tsv.gz`: from `hotspot-detection` module to gather calls that overlap MSKCC hotspots found in any caller (except if only vardict calls the site as variant, we remove these calls since we have a lot of calls unique to vardict which we consider as false positive as discussed [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers#snv-caller-comparison-analysis))
```
Comment (jharenza) [inline analyses/molecular-subtyping-LGAT/README.md]:
```suggestion
* `snv-consensus-plus-hotspots.maf.tsv.gz`: from D3b workflow does a 2/4 consensus plus gather calls that overlap MSKCC hotspots found in any caller
```

**Review by jharenza — APPROVED** [review]
Just updating a few minor things. Can you do one quick rerun with the increased guess_max in case we missed anything? Then you can merge.

**Comment by runjin326** [comment]
@jharenza, re-ran with the guessmax changed and there are some diff in ploidy information when re-ran. Maybe you can do a final check and I can merge?

**Review by jharenza — APPROVED** [review]
I think this should be fine - thanks!

**Pull request merged**